bin/UniProt-Protein-Annotation-NF.R

```diff
@@ -0,0 +1,66 @@
+#!/usr/bin/env Rscript
+
+# Libraries to get annotations using the uniprot web service
+# The docker image on quay.io is out of date, so pull a docker image of
+# bioconductor, and then install
+# Be sure to set the library path to a temporary directory first. 
+dir.create('./Rlibs', showWarnings = FALSE)
+.libPaths('./Rlibs')
+BiocManager::install('UniProt.ws', lib = './Rlibs')
+library(UniProt.ws, lib = './Rlibs')
+
+args = commandArgs(trailingOnly=TRUE)
+
+# Pull out the species names - this is to be specified from the commandline
+spp <- args[1]
+
+# Get the filepath to the protein accessions we'll be mapping. 
+# Also specified from the commandline. 
+ids <- args[2]
+
+# Because this is going to be a fair bit of information, let's make a new directory to house these outputs/gene ontologies. 
+annotDir <- './annotations/'
```

Comment:
This was intended to land in a subdirectory of the publishdir of the local nextflow module that calls it (annotate_uniprot.nf) - specified there as protein-annotations.

So, I was hoping it would land in a directory called params.output/protein-annotations directory, and a second directory called "accessions" in the output dir would house a list of the queried protein accessions.

That directory structure (and returning accessions) is probably not necessary, but actually outlines something I don't totally understand. I would like nextflow to return the specified output and retain that directory structure, but just spits all output files into the publish dir. 

For things like the MCL testing where the independent runs can create duplicate file names that's obviously a problem, but I'm not really sure what the solution is...